src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -28,29 +28,6 @@ public static class NormalizationCatalog
             NormalizingEstimator.NormalizationMode mode = NormalizingEstimator.NormalizationMode.MinMax)
             => new NormalizingEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, inputColumnName ?? outputColumnName, mode);
 
-        /// <summary>
-        /// Normalize (rescale) several columns according to the specified <paramref name="mode"/>.
-        /// </summary>
-        /// <param name="catalog">The transform catalog</param>
-        /// <param name="mode">The <see cref="NormalizingEstimator.NormalizationMode"/> used to map the old values to the new ones. </param>
-        /// <param name="columns">The pairs of input and output columns.</param>
-        /// <example>
-        /// <format type="text/markdown">
-        /// <![CDATA[
-        /// [!code-csharp[Normalize](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs)]
-        /// ]]>
-        /// </format>
-        /// </example>
-        [BestFriend]
-        internal static NormalizingEstimator Normalize(this TransformsCatalog catalog,
```

Comment:
@artidoro made all multiple columns extensions methods internal, and I think he did that for a reason, rather than just deleting them.
I would ask him regarding deleting this internal methods.

Reply:
@Ivanidzo4ka @eerhardt and @TomFinley feel we should remove such methods that are taking enum for normalizer kind because a dedicated method for each normalize kind exists.